pytorch_blade/src/ltc/disc_compiler/passes/cluster.cpp

```diff
@@ -11,14 +11,37 @@
 
 #include "ltc/disc_compiler/passes/cluster.h"
 
+#include <torch/csrc/jit/passes/dead_code_elimination.h>
 #include "compiler/jit/fusion.h"
 #include "compiler/mlir/converters/mhlo_conversion.h"
+#include "ltc/disc_compiler/passes/graph_fuser.h"
 
 #include <torch/script.h>
 namespace torch_disc {
 namespace compiler {
 using namespace ::torch::jit;
 
+bool IsDiscSupports(const torch::jit::Node* node) {
+  if (torch::blade::IsMlirMhloSupported(*node) &&
+      node->kind() != prim::Constant) {
+    for (auto& input : node->inputs()) {
+      // input should be Tensor or Scalar with explict type
+      auto typ = input->type();
+      if (!typ->cast<c10::TensorType>() &&
```

Comment:
By design, do some operations have types other than Tensor and Scalar that will not be supported? Such as, `concat(Tensor[] tensors, int dim=0) -> Tensor`

Reply:
done.